transformer/test_task.py

```diff
@@ -360,6 +360,57 @@ def test_it_supports_put_requests_without_payload(self):
             },
         )
 
+    def test_it_supports_patch_requests_with_payload(self):
+        url = "http://abc.de"
+        r = Request(
+            timestamp=MagicMock(),
+            method=HttpMethod.PATCH,
+            url=urlparse(url),
+            har_entry={"entry": "data"},
+            headers=[Header("a", "b")],
```

Comment:
Version 1.2.0, which was released today, introduced a breaking change - headers are now stored in [`CaseInsensitiveDict`](https://github.com/zalando-incubator/Transformer/blob/master/transformer/request.py#L120), so there's no `Header` class anymore (hence the failing tests as reported in Travis). For the sake of readability it's fine to use a regular Python `dict` in the tests (like in other test cases in this module). 

Thanks for adding the tests! Have you considered adding your username to our [Contributors list](https://github.com/zalando-incubator/Transformer/blob/master/docs/Contributors.rst)? As a maintainer of this project I encourage you to do so 😄

Reply:
Oops, I see I was so 'lucky' to commit almost at the same time 1.2.0 was merged into master.
I have fixed the tests and added myself to the contributors. Thanks again for your help!